docs/LwM2M_Client/STMicroelectronics/STM32L496G-DISCOBG96.md

```diff
@@ -97,9 +99,86 @@ To connect the board:
+in a secure way to prevent unauthorized updates and access to confidential on-device data such as code and
+firmware encryption key.
+
+The **Secure Boot** (Root of Trust services) is immutable code, always executed after a system reset, that checks STM32
+static protections, activates STM32 runtime protections and then verifies the authenticity and integrity of user
+application code before every execution in order to ensure that invalid or malicious code won't be run.
+
+### Additional prerequisites
+- **STM32CubeProgrammer** installed.
+- Python with the following modules: `pycryptodomex`, `ecdsa`, `numpy`, `pyelftools`.
+
+### Binary Compilation
+
+You need to follow a strict compilation order:
+
+0. Compile **SECoreBin** application<br/>
+   This step is needed to create the Secure Engine core binary including all the trusted code and keys mapped inside
+   the protected environment. The binary is linked with the SBSFU code in step 2.
+
+0. Compile **SBSFU** application<br/>
+   This step compiles the SBSFU source code implementing the state machine and configuring the protections. In addition,
+   it links the code with the SECore binary generated at step 1 in order to generate a single SBSFU binary including the
+   SE trusted code.
+0. Compile **UserApp** application (set **Build configuration** to **Release**)<br/>
+   It generates:<br/>
+    1. The user application binary file that is uploaded to the device using the Secure Firmware Update process <br/>
+     (`Projects/STM32L496G-DISCO/UserApp/Binary/Anjay-freertos-client-STM32L496G-[MODEM].sfb`).
+    1. A binary file concatenating the SBSFU binary, the user application binary in clear format, and the corresponding
+     FW header <br/>
+     (`Projects/STM32L496G-DISCO/UserApp/Binary/SBSFU_Anjay-freertos-client-STM32L496G-[MODEM].bin`).
+
+    !!! tip
+        You can set a custom firmware version in the `Application/Inc/default_config.h` file (using `FIRMWARE_VERSION` define).
+        It will be useful when performing FOTA to distinguish the firmware images from each other.
+
+### Flashing
+
+Use **STM32CubeProgrammer** application with `SBSFU_Anjay-freertos-client-STM32L496G-[MODEM].bin` file to program the board (it is advisable to perform **Full chip erase** first). You can open serial port to change default credentials in order to connect to Coiote DM.
+
+After that, you can use Coiote DM to perform firmware update with `Anjay-freertos-client-STM32L496G-[MODEM].sfb` file.
+
+### Disable Secure Protection
+
+!!! important
+    When flashed board with Secure Boot you will need to switch off secure protection to be able to flash the board again.
+
+To deactivate secure application please run **STM32_Programmer_CLI** (Program provided with STM32CubeProgrammer) tool with specific options:
+```
+./<path_to_STM32_Programmer_CLI> -c port=SWD mode=UR -ob RDP=0xBB -ob RDP=0xAA WRP1A_STRT=0xFF WRP1A_END=0x0 -ob displ
+```
+
+ ## Performing firmware update
```

Comment:
Something went wrong here after rendering. I think that the space at the beginning is not needed.

Reply:
Also, I would make it `###` in order to keep it under the `Anjay Client with FOTA` chapter.